docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+services:
+  monitor:
+    image: inzania/network-traffic-metrics:latest
+    network_mode: host
+    environment:
+      - "NTM_INTERFACE=enp5s0"
+      - "NTM_FILTERS=(src net 192.168.1.0/24 and dst net 192.168.1.0/24)"
```

Comment:
Suggestion: these host-specific variables could be [extracted into an `.env` file](https://docs.docker.com/compose/environment-variables/). This way, each user simply needs to insert the appropriate values into the file before running the compose command, instead of modifying the yaml file.

Reply:
Done [here](https://github.com/Pablohn26/network-traffic-metrics/commit/efc61b57218adeb4fbd9d2fd4e54cf77a923a0e9)